app/Models/EntityLog.php

```diff
@@ -32,6 +32,10 @@ class EntityLog extends Model
     public const ACTION_UPDATE = 2;
     public const ACTION_DELETE = 3;
     public const ACTION_RESTORE = 4;
+    public const ACTION_CREATE_POST = 5;
```

Comment:
Missing new @property comments above for auto-completion in vscode/phpstorm (not related to these constants, but can't add a comment on unchanged lines)